ios/RNTPTDocumentView.m

```diff
@@ -103,6 +105,10 @@ - (instancetype)initWithCoder:(NSCoder *)coder
 
 - (void)didMoveToWindow
 {
+    if (@available(iOS 13.0, *)) {
+        self.window.overrideUserInterfaceStyle = self.followSystemDarkMode ? UIUserInterfaceStyleUnspecified : UIUserInterfaceStyleLight;
```

Comment:
```suggestion
        self.viewController.navigationController.overrideUserInterfaceStyle = self.followSystemDarkMode ? UIUserInterfaceStyleUnspecified : UIUserInterfaceStyleLight;
```
I think we can just set it on the internal navigation controller instead of the whole window so that only our viewer is affected by the prop.